nemo_skills/utils.py

```diff
@@ -500,3 +500,44 @@ def resolve_python_module_from_file(py_filepath: str, root_module: str = 'nemo_s
     striped_module = str(stripped_module_path).replace("/", ".")
     striped_module = os.path.splitext(striped_module)[0]
     return striped_module
+
+
+def maybe_get_env(value: Union[int, List[int]], env_name, default=None, cast: Callable[[Any], Any] = None):
```

Comment:
type of value is not correct? Best not to provide type, rather than have incorrect one